src/sentry/runner/commands/run.py

```diff
@@ -426,25 +428,20 @@ def inner(f):
     help="Spawn this many threads to process messages. Defaults to 1.",
 )
 @configuration
-def ingest_consumer(consumer_type, **options):
+def ingest_consumer(consumer_types, **options):
     """
     Runs an "ingest consumer" task.
 
     The "ingest consumer" tasks read events from a kafka topic (coming from Relay) and schedules
     process event celery tasks for them
     """
-    from sentry.ingest.ingest_consumer import ConsumerType, get_ingest_consumer
+    from sentry.ingest.ingest_consumer import get_ingest_consumer
     from sentry.utils import metrics
 
-    if consumer_type == "events":
-        consumer_type = ConsumerType.Events
-    elif consumer_type == "transactions":
-        consumer_type = ConsumerType.Transactions
-    elif consumer_type == "attachments":
-        consumer_type = ConsumerType.Attachments
-
-    with metrics.global_tags(ingest_consumer_type=consumer_type, _all_threads=True):
-        get_ingest_consumer(consumer_type=consumer_type, **options).run()
+    with metrics.global_tags(
+        ingest_consumer_types=",".join(sorted(consumer_types)), _all_threads=True
+    ):
+        get_ingest_consumer(consumer_types=consumer_types, **options).run()
```

Comment:
Shouldn't you convert `consumer_types` to a list of `ConsumerType` instances?

Reply:
The old code didn't actually do any mapping at all given the enum variants were the same exact strings. I cleaned this up a little